### PR TITLE
Removing bad characters from the messages

### LINKED ISSUE #14449: Bad Characters in SameSite Warning Messages
The single quote characters (') in some of the warning messages for the SameSite feature, are being treated as bad characters. For consistency, these bad characters and other single quotes should be removed.

### PR DESCRIPTION
**Pull request by mrsaldana**
fixes #14449

### CODE REVIEW

**Comment by mrsaldana** [comment]
#build

**Review by billw1024 — APPROVED** [review]
looks good

**Comment by mrsaldana** [comment]
#build

File: dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops [hunk 1]
```diff
@@ -166,14 +166,14 @@ decompression.tolerance.reached=CWWKT0034E: The HTTP channel attempted to decomp
...
 decompression.tolerance.reached.useraction=When this exception happens on a valid request body, reconfigure the decompressionTolerance and decompressionRatioLimit httpOption attributes to achieve the appropriate decompression results.
 
-cookies.samesite.duplicateName=CWWKT0035W: A duplicate cookie name or pattern [{0}] was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored for all configuration lists. Any cookie name or pattern that is defined by the ��lax��, ��none��, and ��strict�' configurations can be defined in only one of the three configurations.
+cookies.samesite.duplicateName=CWWKT0035W: A duplicate cookie name or pattern [{0}] was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored for all configuration lists. Any cookie name or pattern that is defined by the lax, none, and strict configurations can be defined in only one of the three configurations.
```
Comment (ManasiGandhi) [inline dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops:169]: Change,
" A duplicate cookie name or pattern [{0}] was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored for all configuration lists. Any cookie name or pattern that is defined by the lax, none, and strict configurations can be defined in only one of the three configurations."
to
" A duplicate [{0}]cookie name or pattern  was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored for all configuration lists. Any cookie name or pattern that is defined by the lax, none, and strict configurations can be defined in only one of the three configurations."

File: dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops [hunk 2]
```diff
@@ -166,14 +166,14 @@ decompression.tolerance.reached=CWWKT0034E: The HTTP channel attempted to decomp
...
+cookies.samesite.unsupportedWildcard.useraction=Ensure that only a single wildcard (*) character is used per pattern in the lax, none, and strict lists. Additionally, use of this character is supported only under the following two conditions: as a stand-alone value or following cookie name prefixes.
 
-cookies.samesite.knownDuplicateName=CWWKT0037W: A cookie name or pattern [{0}], which is marked as a duplicate, was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored. Any cookie name or pattern that is defined by the ��lax��, ��none��, and ��strict�' configurations can be defined in only one of the three configurations.�
+cookies.samesite.knownDuplicateName=CWWKT0037W: A cookie name or pattern [{0}], which is marked as a duplicate, was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored. Any cookie name or pattern that is defined by the lax, none, and strict configurations can be defined in only one of the three configurations.
```
Comment (ManasiGandhi) [inline dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops:177]: Change,
"A cookie name or pattern [{0}], which is marked as a duplicate, was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored. Any cookie name or pattern that is defined by the lax, none, and strict configurations can be defined in only one of the three configurations."
to,
"A  [{0}] cookie name or pattern, which is marked as a duplicate, was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored. Any cookie name or pattern that is defined by the lax, none, and strict configurations can be defined in only one of the three configurations."

**Review by ManasiGandhi — COMMENTED** [review]
Some edit suggestions.

File: dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops (hunk 1 above)
Comment (mrsaldana) [reply dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops:169]: Discussed internally through Slack, marking this as resolved.

File: dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops (hunk 2 above)
Comment (mrsaldana) [reply dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops:177]: Discussed internally through Slack, marking this as resolved.

**Comment by mrsaldana** [comment]
#run-libby-bot

**Comment by skasund** [comment]
L2 message review completed

**Pull request merged**